### PR TITLE
query-multiple-tables

### PR DESCRIPTION
**Pull request by Kgomotso196**
# PROJECT TO DOs
### IN THIS PROJECT I:
1. Used primary key & foreign key mechanism for joining tables.
2. Queried multiple tables.
3. Prepared complex queries that answer analytical questions.
4. Made sure that I used correct ([Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)).
5. Made sure that I documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
6.  Provided screenshots for reference:
d1
d2
d3
d4
d5
d6
d7

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,55 +1,116 @@
...
-SET species = 'digimon'
-WHERE name LIKE '%mon';
-UPDATE animals
-SET species = 'pokemon'
-WHERE species IS NULL;
-SELECT * FROM animals;
+
+    UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+    UPDATE animals SET species = 'pokemon' WHERE species is null;
```
Comment (LionRouge1) [inline queries.sql]: - [ ] As requested in the project requirement, everything that populates tables with data should be saved in the `data.sql`. Therefore I suggest you move these codes into the `data.sql` file to follow the project requirement.

File: queries.sql [hunk 2]
```diff
@@ -1,55 +1,116 @@
...
 COMMIT;
-SELECT * FROM animals;
+SELECT species from animals;
 
 BEGIN;
-DELETE FROM animals
+
+    DELETE FROM animals;
+
+SELECT species from animals;
```
Comment (LionRouge1) [inline queries.sql]: - [ ] same thing here as pointed out earlier:point_up_2:

File: queries.sql [hunk 3]
```diff
@@ -1,55 +1,116 @@
...
+
+    DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+    SAVEPOINT SP1;
+
+    UPDATE animals SET weight_kg = weight_kg * -1;
+
+    ROLLBACK TO SP1:
+
+    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
```
Comment (LionRouge1) [inline queries.sql]: - [ ] same thing here as pointed out earlier:point_up_2:

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @Kgomotso196, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Set up one-to-many relationships with foreign keys :heavy_check_mark: 
- Use JOIN to query multiple tables :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Kgomotso196) [reply queries.sql]: @LionRouge1, thank you for a constructive feedback. Working on it.

**Review by DeliceLydia — APPROVED** [review]
## Hi @Kgomotso196, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed changes requested



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**